recipes/advbumpversion/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "advbumpversion" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a605d545ca4153c38b82c994729b1bc11b7d23bff4226e45440bbbc1881df8a5
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - bumpversion = bumpversion:main
+    - bump2version = bumpversion:main
+    - advbumpversion = bumpversion:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  # Cannot run tests since they're not installed with the package and
+  # conda-build runs tests in an isolated environment
+  # requires:
+  #   - pytest
+  #   - mock
+  # commands:
+  #   - py.test
+  imports:
+    - bumpversion
+
+about:
+  home: https://github.com/andrivet/advbumpversion'
```

Comment:
Does the `home` line have a dangling quotation mark?

Reply:
Fixed